e2e/cases/server/ssr/src/index.server.tsx

```diff
@@ -3,6 +3,8 @@ import ReactDOMServer from 'react-dom/server';
 import App from './App';
 import { assert } from './assert.server';
 
+console.log('load ssr');
```

Comment:
Make the case correct

```suggestion
console.log('load SSR');
```